### PR TITLE
Allow to fetch union data properties in unified way

### PR DESCRIPTION
**Pull request by Jampire**
This PR came from the discussion #379 . It allows to fetch lazy and non-lazy properties in the same way (regardless the fact how DTO was created - lazy or not).

### CODE REVIEW

File: tests/DataTest.php [hunk 1]
```diff
@@ -2278,3 +2279,31 @@ public function __construct(
     expect($invaded->_except)->toBeEmpty();
     expect($invaded->_wrap)->toBeNull();
 });
+
+it('can fetch lazy union data', function () {
+    $data = UnionData::from(1);
+
+    expect($data->id)
```
Comment (freekmurze) [inline tests/DataTest.php:2286]: I would make separate expectations. Imho the `and` makes it less readable to see what is actually being tested.

**Comment by rubenvanassche** [comment]
Looking good, ping me if you have updated the tests like Freek asked and then I'll tag a new release

**Comment by Jampire** [comment]
> Looking good, ping me if you have updated the tests like Freek asked and then I'll tag a new release

@rubenvanassche , it's updated.

**Comment by rubenvanassche** [comment]
Thanks!

**Comment by francoism90** [comment]
https://github.com/spatie/laravel-data/pull/354

Hmm, I hope this fixes this on Livewire as well. :)

**Pull request merged**